app/components/works/edit/work_subtypes_component.rb

```diff
@@ -5,7 +5,7 @@ module Edit
     # Component for the work subtypes field
     class WorkSubtypesComponent < ApplicationComponent
       def initialize(form:, work_type:, label:, tooltip: nil, more_options: false, minimum_terms: 0, # rubocop:disable Metrics/ParameterLists
-                     error_field_name: nil, required_subtypes: [])
+                     error_field_name: nil, required_subtypes: [], help_text: nil)
```

Comment:
Having this argument carry the help text suggests that we have different values of help text but we don't seem to. It may be clearer for the help text to live in this class and to change this param to `display_help_text: false`. And then change the injected help text params above in the collection work types component to `display_help_text: true`.

Reply:
It has been our pattern to inject labels, help text, etc. in "generic" components. This seems consistent with that pattern.

I think part of the confusion is that `Works::Edit::WorkSubtypesComponent` should be moved to `Edit::WorkSubtypesComponent` since it is used both for works and collections.

Reply:
On it.